File: cpp/215.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        vector<int> f(20001);
        for (int i : nums)
            f[i + 10000]++;
        for (int i = 20000; i >= 0; i--) {
            k -= f[i];
            if (k <= 0)
                return i - 10000;
        }
        return 0;
    }
};

File: python3/215.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        f = [0] * 20001
        for i in nums:
            f[i + 10000] += 1
        for i in range(20000, -1, -1):
            k -= f[i]
            if k <= 0:
                return i - 10000
        return 0
